src/front/glsl/parser.rs

```diff
@@ -1100,6 +1100,16 @@ pomelo! {
                     for (id, initializer) in d.ids_initializers {
                         let init = initializer.map(|init| extra.solve_constant(init.expression)).transpose()?;
 
+                        // use StorageClass::Handle for texture and sampler uniforms
+                        let class = if storage_class == StorageClass::Uniform {
+                            match &extra.module.types[d.ty].inner {
```

Comment:
please no matching on references

Reply:
hmmm, think I was struggling with the syntax, and along the way the compiler suggested me to add the `&` ... still a noob :)